osh/hub/settings.py

```diff
@@ -56,7 +56,7 @@
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
         'DIRS': [
             # `/etc/osh/hub/templates` overrides `index-content.html`
-            os.path.join("etc", "osh", "hub", "templates"),
+            os.path.join("/etc/osh/hub/templates"),
```

Comment:
TBH, calling `os.path.join` with a single argument is completely useless.

Reply:
if we want to use L59, why not use `"/etc/osh/hub/templates"` directly, or use `os.path.join("/", "etc", "osh", "hub", "templates")`

Reply:
https://github.com/openscanhub/openscanhub/pull/264/files